crates/core/src/config.rs

```diff
@@ -122,12 +123,21 @@ impl ConfigPaths {
             fs::create_dir_all(db_dir.join("local"))?;
         }
 
+        let event_log = app_data_dir.join("_EVENT_LOG");
```

Comment:
Is there any point in persisting the event log given that it won't be useful after a restart due to fresh connections?

Reply:
This logs more than route events and will be used for debugging.